_posts/2019-01-06-DRQN.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title:  "Deep Recurrent Q-Network"
+date:   2019-01-06 11:42:04 +0530
+categories: jekyll update
+author: Madhuparna Bhowmik
+---
+
+In this article, we will learn about Deep recurrent Q-learning and POMDP and find out why DRQN works better in case of POMDP than DQN.
+
+## MDP vs. POMDP (Partially Observable MDP)
+
+In an MDP we have full knowledge of the environment. But in most of the cases, it is not possible to know the complete state of the environment. Like a single game screen is not sufficient to determine the state of the system. For example, have a look at the picture below -
+
+![Passing ball](/assets/passing_ball.png){:height="50%" width="50%"}
+
```

Comment:
The height and width command is being shown in the blog. Replace it with an alternate way to adjust height and width

Reply:
Just remove the height and width part. It's not required.